### PR TITLE
PBF-733 : Audio not persisting for youtube plugin

### PR DESCRIPTION
**Pull request by mitulparekh**

### CODE REVIEW

File: src/youtube/js/youtube.js [hunk 1]
```diff
@@ -119,15 +119,19 @@ require("../../../html5-common/js/utils/constants.js");
...
+      {
+        case "play":
+          element.play();
+          hasPlayed = true;
+          break;
+        case "seek":
+          element.seek(javascriptCommandQueue[i][1]);
+          break;
+        case "setVolume":
+          element.setVolume(javascriptCommandQueue[i][1]);
```
Comment (aeng7) [inline src/youtube/js/youtube.js]: Can we add a length/null check to ensure that javascriptCommandQueue[i][1] exists?

File: src/youtube/js/youtube.js [hunk 2]
```diff
@@ -119,15 +119,19 @@ require("../../../html5-common/js/utils/constants.js");
...
-        element.play();
-        hasPlayed = true;
+      switch(javascriptCommandQueue[i][0])
+      {
+        case "play":
+          element.play();
+          hasPlayed = true;
+          break;
+        case "seek":
+          element.seek(javascriptCommandQueue[i][1]);
```
Comment (aeng7) [inline src/youtube/js/youtube.js]: Can we add a length/null check to ensure that javascriptCommandQueue[i][1] exists?

File: src/youtube/js/youtube.js [hunk 3]
```diff
@@ -119,15 +119,19 @@ require("../../../html5-common/js/utils/constants.js");
     playerReady = true;
     for(var i = 0; i < javascriptCommandQueue.length; i++) 
     {
-      if(javascriptCommandQueue[i][0] === "play")
-      { 
-        element.play();
-        hasPlayed = true;
+      switch(javascriptCommandQueue[i][0])
```
Comment (aeng7) [inline src/youtube/js/youtube.js]: Can we add a length/null check to ensure that javascriptCommandQueue[i][0] exists?

Comment (dchauOoyala) [inline src/youtube/js/youtube.js]: The `javascriptCommandQueue[i][0]` will throw an error if `javascriptCommandQueue` contains bad values like `null` or `undefined`. But it does not seem like 'javascriptCommandQueue` can ever be populated with those type of values, so these checks will suffice.

**Comment by dchauOoyala** [comment]
LGTM

**Pull request merged**